codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -460,6 +468,94 @@ picToObj' pic =
                 ]
                 obj
             retVal obj
+        Polygon cs pts w closed smooth -> do
+            obj <- init "path"
```

Comment:
The names here can be changed to match the functions: `polygon`, `thickPolygon`, etc.